recipes/mikktspace/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+
+class MikkTSpaceConan(ConanFile):
+    name = "mikktspace"
+    description = " A common standard for tangent space used in baking tools to produce normal maps."
+    homepage = "https://github.com/mmikk/MikkTSpace"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
```

Comment:
it's not an MIT license.... I think this is a personal one =? I didnt find a match https://opensource.org/licenses/alphabetical

Reply:
This is the zlib license

https://spdx.org/licenses/Zlib.html

```suggestion
    license = "Zlib"
```